WalletWasabi.Gui/Rpc/WasabiJsonRpcService.cs

```diff
@@ -121,18 +121,27 @@ public object GetStatus()
 		}
 
 		[JsonRpcMethod("build")]
-		public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int feeTarget, string password = null)
+		public string BuildTransaction(PaymentInfo[] payments, OutPoint[] coins, int feeTarget = 0, decimal feeRate = 0, string password = null)
```

Comment:
Is this breaking compatibility? 

- If someone used this like BuildTransaction(paments, coins, 7, "mypassword"), it won't work, will it? You changed the order of the args. 
- Check if Nullable changed the evaluation of default parameters. Password might become a compulsory parameter if you don't put ?(nullable) there.

Reply:
You're right. I didn't see this.